src/lib.rs

```diff
@@ -11,14 +11,16 @@ pub enum GitHost {
 }
 
 impl GitHost {
-    pub fn source(&self, package: &Package) -> String {
+    pub fn source(&self, package: &Package, no_bin: bool) -> String {
+        // Expecting binary tarballs to be uploaded with a platform identifier.
+        let platform_identifier = if no_bin { "" } else { "-x86_64" };
         match self {
             GitHost::Github => format!(
-                "{}/releases/download/v$pkgver/{}-$pkgver-x86_64.tar.gz",
+                "{}/releases/download/v$pkgver/{}-$pkgver{platform_identifier}.tar.gz",
```

Comment:
Style: Since we're already using `{}` in two places, let's do the same for `platform_identifier`.

Reply:
actioned